I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 7, in script
    jump chapter1_start
ScriptError: could not find label 'chapter1_start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 7, in script
    jump chapter1_start
  File "/Users/mykyhuynh/Documents/renpy-8.1.3-sdk/renpy/ast.py", line 1992, in execute
    rv = renpy.game.script.lookup(target)
  File "/Users/mykyhuynh/Documents/renpy-8.1.3-sdk/renpy/script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'chapter1_start'.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.1.3.23091805
Otome Ongoing 1.0
Sat Mar  1 12:48:16 2025
